native/golang/cmds/uss.go

```diff
@@ -239,14 +239,16 @@ func HandleWriteFileRequest(conn *utils.StdioConn, params []byte) (result any, e
 		}
 	}
 
-	tokens := strings.Fields(strings.TrimSpace(string(out)))
-	etag := ""
-	created := false
-	if len(tokens) > 0 {
-		etag = tokens[0]
+	output := utils.YamlToMap(string(out))
+
+	var etag string
+	if etagValue, exists := output["etag"]; exists {
+		etag = fmt.Sprintf("%v", etagValue)
 	}
-	if len(tokens) > 1 && tokens[1] == "true" {
-		created = true
+
+	var created string = "false" // default value
+	if createdValue, exists := output["created"]; exists {
+		created = fmt.Sprintf("%v", createdValue)
 	}
```

Comment:
"Created" is always a boolean value (`true` or `false`) in this context - I think we should keep it as a `bool` in the Go types, and update the logic to convert the value the command returns to the right boolean value.